### PR TITLE
Api

### PR DESCRIPTION
**Pull request by ijeomaodinko**
let the program be able to add and remove book features directly in your application using Redux and React state. Now you should refactor your code to use the Bookstore API to add and remove books.

### CODE REVIEW

**Review by Ismailco — COMMENTED** [review]
Hi @ijeomaodinko, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no documentation, kindly use the [README template](https://github.com/microverseinc/readme-template/blob/master/README.md) provided by microverse to build your README.md file

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/redux/books/books.js [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+// to add action types
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+const READ_BOOKS = 'bookstore/books/READ_BOOK';
+
+// to get the url
+const apiUrl = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/EHbt3DaBZgT4pqfPpOLB/books/';
+
+// the get book action
+export const getBookThunk = createAsyncThunk(READ_BOOKS, async () => {
```
Comment (mateo951) [inline src/redux/books/books.js:15]: `createAsyncThunk` is used in the project for creating thunk action creators 🚀 Well done!

File: src/redux/books/books.js [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+  const response = await axios.get(`${apiUrl}`).catch((error) => { console.log(error); });
```
Comment (mateo951) [inline src/redux/books/books.js:16]: All [Bookstore API](https://www.notion.so/Bookstore-API-51ea269061f849118c65c0a53e88a739) calls are done with `fetch` OR `axios` ✔

**Review by mateo951 — APPROVED** [review]
### Hi @ijeomaodinko, 

Your project is complete!  There is nothing else to say other than... it's time to merge it 🐐 
### Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**